lib/features/email/presentation/extensions/calendar_event_extension.dart

```diff
@@ -269,4 +269,20 @@ extension CalendarEventExtension on CalendarEvent {
       return '';
     }
   }
+
+  List<String> get videoConferences {
+    if (extensionFields != null && extensionFields!.mapFields.isNotEmpty) {
+      final videoConferences = extensionFields!.mapFields['X-OPENPAAS-VIDEOCONFERENCE'];
+      if (videoConferences != null) {
+        final videoConferencesNotEmpty = videoConferences
+          .whereNotNull()
+          .where((link) => link.isNotEmpty)
```

Comment:
do you think `.where((link) => link != null && link.isNotEmpty)` faster ?

Reply:
It's not faster. Above all, the result you get is a list of objects that can be `nullable` during compile time.